src/script/lua_api/l_client.cpp

```diff
@@ -36,6 +36,20 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "util/string.h"
 #include "nodedef.h"
 
+#define checkCSMRestrictionFlag(flag) \
+	( getClient(L)->checkCSMRestrictionFlag(CSMRestrictionFlags::flag) )
+
+// FIXME: This should eventually be moved somewhere else
```

Comment:
Another note referring to this struct array in `networkprotocol.h` would be great so that they stay synced.